### PR TITLE
issue/96

### PR DESCRIPTION
**Pull request by comamitc**
Fixed issue #96.  Weird branch name because I botched a previous branch and deleted it.

**note:** last commit message should read 96-4 & 96-5

Perfect project:
screen shot 2015-02-25 at 3 14 11 pm

Duplicate path or output directory:
screen shot 2015-02-25 at 3 13 53 pm

Malformed project.clj:
screen shot 2015-02-25 at 3 14 22 pm

### CODE REVIEW

File: src-cljs/cuttle/projects.cljs [hunk 1]
```diff
@@ -36,15 +37,42 @@
...
-           :cljsbuild cljsbuild
-           :filename filename
-           :name (name (nth prj1 1))
-           :version (nth prj1 2))))
+        prj1 (try-read-string contents)]
+        (if (contains? prj1 :error)
+          ;; then
+          (assoc prj1
+                 :filename filename
+                 :name filename
```
Comment (shaunlebron) [inline src-cljs/cuttle/projects.cljs]: this be good

File: .nrepl-port [hunk 2]
```diff
@@ -0,0 +1 @@
+54840
```
Comment (oakmac) [inline .nrepl-port]: Is this something that should be added to the repo or possibly added to .gitignore? Disclaimer - I'm not 100% sure what this is for; something involving a REPL I assume.

Comment (shaunlebron) [inline .nrepl-port]: yeah, that should go away.  I'll just delete it after the merge.

Comment (comamitc) [inline .nrepl-port]: You know, I didn't notice this file in my `git status`. I think this is a .gitignore thing. I will remove and push.

**Comment by shaunlebron** [comment]
Thanks Mitch, looks awesome

File: src-cljs/cuttle/util.cljs [hunk 3]
```diff
@@ -19,6 +20,16 @@
 ;;------------------------------------------------------------------------------
 ;; Util Functions
 ;;------------------------------------------------------------------------------
+(defn not-empty?
```
Comment (oakmac) [inline src-cljs/cuttle/util.cljs]: I'm not crazy about this function. Feels like it should be equivalent to `(not (empty? my-coll))` and just used in that way directly in the code.

**Pull request merged**